data/json/monsters/insect_spider.json

```
@@ -1788,5 +1788,36 @@
     "death_function": [ "NORMAL" ],
     "special_attacks": [ { "type": "leap", "cooldown": 4, "max_range": 4, "allow_no_target": true }, [ "EAT_CROP", 120 ] ],
     "flags": [ "SEES", "HEARS", "SMELLS", "CLIMBS", "LARVA", "STUMBLES", "PATH_AVOID_FIRE" ]
+  },
+  {
+    "id": "mon_stag_beetle",
+    "type": "MONSTER",
+    "name": { "str": "Huge Stag Beetle" },
+    "description": "Once a priceless bug, it has grown to the size of a calf.  Although you doubt that it's going to attack you, those large pincers could probably break your leg.",
+    "default_faction": "stag_beetle",
+    "bodytype": "flying insect",
+    "species": [ "INSECT" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 50,
+    "speed": 110,
+    "material": [ "flesh" ],
+    "symbol": "B",
+    "color": "black",
+    "aggression": 0,
+    "morale": 50,
+    "melee_skill": 4,
+    "melee_dice": 1,
+    "melee_dice_sides": 8,
+    "armor_bash": 12,
+    "armor_cut": 14,
+    "armor_bullet": 10,
+    "dodge": 1,
+    "melee_cut": 0,
+    "vision_day": 10,
+    "harvest": "mammal",
```

```suggestion
    "harvest": "arachnid",
```
Unless you want to add a custom harvest group, the generic arachnid fits them well enough.